ruby/vampires.rb

```diff
@@ -0,0 +1,80 @@
+CURRENT_YEAR = 2017
```

Comment:
Try to avoid using constant variables in your code.  I understand why you chose this and I do think it shows great intuition.  That said, constants are dangerous because any change to them could cause lots of issues when working with many files in the future.

A more dynamic and scalable approach would be to use the Ruby `Time` class.

Checkout `Time.new.year`